client/receiver.go

```diff
@@ -113,9 +113,14 @@ func request(url string, method string, headers HeaderOption, input interface{},
 	return nil
 }
 
-type receiver struct{}
+type receiver struct {
+	Auth *NoAuthOptions
+}
 
-func (*receiver) Recv(url string, method string, input interface{}, output interface{}) error {
+func (r *receiver) Recv(url string, endPoint string, method string, input interface{}, output interface{}, in ...string) error {
+	url := strings.Join([]string{
+		endPoint,
+		urls.GenerateURL(resource, urls.Client, k.Auth.GetTenantId(), in...)}, "/")
```

Comment:
Can you explain where are `resource`, `urls` and `k` come from?

Reply:
I resubmitted pr, please review again.